assets/scripts/Map/Bridge/blockPlatform.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+const { ccclass, property } = cc._decorator;
```

Comment:
```suggestion
const { ccclass } = cc._decorator;
```
O property não está sendo usado pra nada, remova-o